NAMESPACE

```diff
@@ -8,8 +8,13 @@ export(toolCubicFunctionAggregate)
 export(toolCubicFunctionDisaggregate)
 export(toolFillEU34Countries)
 export(toolSolarFunctionAggregate)
+import(GDPuc)
+import(edgeTransport)
 import(madrat)
 import(magclass)
+import(mrcommons)
+import(mrdrivers)
+import(mrindustry)
```

Comment:
is this in lie with the input data generation environment for EDGE-T? @ahagen-pik fine for you?